form/src/main/java/org/dhis2/form/ui/Form.kt

```diff
@@ -90,14 +89,6 @@ fun Form(
                 items = sections,
                 key = { _, fieldUiModel -> fieldUiModel.uid },
             ) { _, section ->
-                val isSectionOpen = remember(section.state) {
-                    derivedStateOf { section.state == SectionState.OPEN }
-                }
-
-                LaunchIfTrue(isSectionOpen.value) {
-                    scrollState.animateScrollToItem(sections.indexOf(section))
-                    focusManager.moveFocusNext(focusNext)
-                }
```

Comment:
I may be wrong and we can discussed this with the team, but I think this should optional and not be removed completely

Reply:
Do you mean configurable the ASWA? I think that should be addressed in a different issue if so.

Reply:
I think that this is for all cases. 
There is a use case where we need this for, but with the current form is broken. It's for error handling, when user taps on sync conflicted item it should navigate to the form and scroll to the selected item, as it is not working maybe we can remove this for now and think about how to do it when we fix the other issue: 
![image](https://github.com/dhis2/dhis2-android-capture-app/assets/3330344/6058825d-00a4-4ddb-84aa-d924a5181656)

Reply:
I did take this into account but as Andres mentions it's not working currently with new form, and the part that I have removed checks for first open section, rather than any error field. I will create an issue to address the sync conflict item.

Reply:
Maybe I'm thinking far ahead. Since we're planing to release the form library at some point, would be like to stop automatically scrolling to an open section on all forms?.

Reply:
Maybe we could have a conversation with design we want to allow this or not. In this issue I could maybe add it in as a variable similarly to how we did it with the useCompose one. @marcamsn  What do you think?